README.md

```diff
@@ -35,6 +37,35 @@ Enter [*Docker Content Trust (DCT)*](https://docs.docker.com/engine/security/tru
 
 **Connaisseur** solves this problem by bringing image signature verification into Kubernetes. It uses the already existing Docker Content Trust solution and adds it to the cluster as a mutating admission controller. It pulls trust data from *Notary* and verifies it before any commitment to Kubernetes happens.
 
+## Demo
+
+We provide two public sample images for testing. Setting the trust anchor of Connaisseur to our public key allows you to have a working demo in just a few minutes. Below you see a full setup starting [MicroK8s](https://microk8s.io/) as a test cluster, installation of Connaisseur and testing with a signed and unsigned image.
```

Comment:
Would put a comma after below. Afaik its necessary if there could be confusion, and it could read "below, you see" or "below you, see"

Reply:
"MicroK8s as a test cluster" sounds weird, why not "a Micro test cluster?